src/amo/components/LandingPage/styles.scss

```diff
@@ -10,17 +11,14 @@
 }
 
 .LandingPage-header {
-  margin: 30px 20px 20px;
+  background-color: $white;
```

Comment:
This can all go away (except maybe the `margin-bottom`) if this becomes a `<Card>`.